src/main/java/com/kajan/iworkflows/service/impl/LearnOrgServiceImpl.java

```diff
@@ -25,33 +26,51 @@
 public class LearnOrgServiceImpl implements LearnOrgService {
     private final OauthTokenService oauthTokenService;
     private final RestTemplate restTemplate;
-
-    private final String userInforUri;
+    private final String userInfoUri;
+    private final String username;
+    private final String password;
+    private final String HEADER_VALUE_BASIC = "Basic ";
 
     @Autowired
-    public LearnOrgServiceImpl(@Value("${learnorg.uri.userinfo}") String userInforUri, OauthTokenService oauthTokenService, RestTemplate restTemplate) {
+    public LearnOrgServiceImpl(@Value("${learnorg.uri.userinfo}") String userInforUri,
+                               OauthTokenService oauthTokenService, RestTemplate restTemplate,
+                               @Value("${iworkflows.credentials.learnorg.username}") String username,
+                               @Value("${iworkflows.credentials.learnorg.password}") String password) {
         this.oauthTokenService = oauthTokenService;
         this.restTemplate = restTemplate;
-        this.userInforUri = userInforUri;
+        this.userInfoUri = userInforUri;
+        this.username = username;
+        this.password = password;
     }
 
     @Override
     public UserStore getLearnOrgUserInfo(Principal principal) {
         TokenDTO tokenDTO = this.oauthTokenService.getToken(principal.getName(), LEARNORG);
         String accesstoken = tokenDTO.getAccessToken().getValue();
 
-        HttpHeaders headers = new HttpHeaders();
-        headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
         MultiValueMap<String, String> map = new LinkedMultiValueMap<String, String>();
         map.add("access_token", accesstoken);
-        HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<MultiValueMap<String, String>>(map, headers);
-        log.debug("uri : " + userInforUri);
-        ResponseEntity<UserStore> response = this.restTemplate.postForEntity(userInforUri, request, UserStore.class);
+        HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<MultiValueMap<String, String>>(map, getLearnOrgHeaders());
+        log.debug("uri : " + userInfoUri);
```

Comment:
don't use +, instead, use interpolation like `log.debug("uri : {} ", userInforUri)`, {} will be automatically replaced by userInfoUri

Reply:
ok